tsconfig.json

```diff
@@ -32,6 +33,8 @@
     "src/**/*"
   ],
   "exclude": [
-    "src/*/files/**/*"
+    "src/*/files/**/*",
```

Comment:
what is this `src/*/files` folder?

Reply:
it was there before, not sure why it flags this as a change.. I think it is a remnant from when there was a sandbox environment.. it's a @shmool question :)

Reply:
Added comma in the end of the line...

Reply:
The file was generated by the schematics CLI this way